_posts/2019-12-24-templatowanie-jobow-jenkinsa.md

```diff
@@ -0,0 +1,114 @@
+---
+layout:    post
+title:     "Automat dodający joby do Jenkinsa"
+published: false
+date:      2019-12-24 08:00:00 +0100
+author:    dkubiak
+tags:
+    - jenkins job
+    - jenkins template
+    - devops
+    - jenkins-job-builder
+    - jenkins-jobs
+---
+## Zastosowanie
+
+W ogarniającym nas świecie mikroserwisów skala projektów do utrzymania staje sie ogromna. Każdy z tych projektów musimy przecież: zbudować, przetesować, zdeployować itd. Przy liczbie projektów 20+ przestaje to być trywialne. W tym artukule zajmniemy się pierwszym zagadnieniem budowaniem, ale opisany tutaj sposób bez problemu można zastosować do innych aspektów.
+
+Do budowania naszych projektów 20+ zazwyczaj staramy sie utrzymać jedno narzędzie i to oczywiście działa na plus. W tym artykule posługiwać będziemy się mavenem.
```

Comment:
usunąłbym 8 sztuk nasz, nasze, naszego - imo brzmi to lepiej

Reply:
nie wiem o co biega :/

Reply:
Zamiast "Do budowania naszych projektów" lepiej brzmi "Do budowania projektów". We wpisie jest 8x "nasz" w różnych odmianach.

Reply:
OMG 🥇

Reply:
done!